fuse/osxfuse/Portfile

```diff
@@ -119,6 +119,15 @@ post-extract {
             move ${workpath}/osxfuse-${mp.comp}-${mp.rev} ${workpath}/${worksrcdir}/${mp.comp}
         }
     }
+
+    if {${os.major} == 17} {
+        ui_msg "If building this port fails, consider applying the following workaround:"
+        ui_msg ""
+        ui_msg "    cd \$(xcode-select -p)/Toolchains"
+        ui_msg "    sudo ln -s XcodeDefault.xctoolchain OSX10.13.xctoolchain"
+        ui_msg ""
+        ui_msg "See https://trac.macports.org/ticket/54939 for more information."
+    }
```

Comment:
How about only displaying this message if the user has not already performed the workaround?

Reply:
Good idea! Added in https://github.com/macports/macports-ports/pull/1749/commits/015dedfce52f782aa56a074dac115605561c60e9